src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int Run(int maxTimeDiff, char *aavild_data) {
+
+    time_t cur_time; 
+    //char * aavild_data = "20211122080000";
+    struct tm timeptr;
+
+    cur_time = time(NULL);
+    printf("Current timestamp = %ld\n", cur_time);
+
+    strptime(aavild_data, "%Y%m%d%H%M%S", &timeptr);
+    time_t aav_time = mktime(&timeptr);
```

Comment:
aav_time is definitely not a good name here. Im not sure at all what this variable is even supposed to be.

Reply:
I will make a comment explaining this